test/test_docker.py

```diff
@@ -210,14 +214,14 @@ def test_run_args_without_platform_config(self, fixtures):
         }),
         'image_id': image_id(),
         'project_root': filepath(),
-        'platform_config_path': one_of(filepath(), none()),
+        'platform_config_paths': lists(elements=filepath(), max_size=1),
```

Comment:
Why do we have this with `max_size=1` always?

Reply:
I was trying to maintain the original intent of this test (i.e. 0 or 1), but I guess it should work with more.

Reply:
Probably worth expanding the test then